packages/website/docs/standard-library/rest/decorators.md

```diff
@@ -279,6 +280,16 @@ Syntax:
 `@segmentSeparator` defines the separator string that is inserted between the target's `@segment` and the preceding route path in auto-generated routes.
```

Comment:
We should update the docs if we are deprecating this (and telling people what to use instead)

Reply:
I agree... I just didn't see any examples of us deprecating a decorator.

Reply:
@allenjzhang are there any examples of how I should document this?